pkg/server/proto/server.proto

```diff
@@ -485,6 +485,12 @@ service Waypoint {
 
   // List releases for a given application.
   rpc UI_ListReleases(UI.ListReleasesRequest) returns (UI.ListReleasesResponse);
+
+  //List UI events for a given application.
+  rpc UI_ListEvents(UI.ListEventsRequest) returns (UI.ListEventsResponse);
```

Comment:
I think you need an entry in `gateway.yml` to go along with this

Reply:
I just added that in.